src/main/java/Pages/ConfigPage.java

```diff
@@ -0,0 +1,36 @@
+package Pages;
```

Comment:
Package names are written in all lower case to avoid conflict with the names of classes or interfaces.